project.clj

```diff
@@ -55,15 +55,15 @@
                                     :password      :env/clojars_password
                                     :sign-releases false}]]
   :pedantic? :warn
-  :java-source-paths ["src/com"]
+  :java-source-paths ["src/com" "test/com"]
```

Comment:
Why this change? We don't package tests in the source.

Reply:
Makes sense. I will move it to test profile.

Reply:
Done